tests/e2e/exportCustodialUserProfileUnitsMMOLLFileTypeJson.js

```diff
@@ -0,0 +1,97 @@
+    const clinicianUsername = browser.globals.clinicianUsername;
+    const clinicianPassword = browser.globals.clinicianPassword;
+    const clinicWorkspacePage = browser.page.clinicWorkspacePage();
+    const clinicWorkspace = clinicWorkspacePage.section.clinicWorkspace;
+    loginPage.loadPage();
+    loginPage.userLogin(clinicianUsername, clinicianPassword);
+    clinicWorkspace.expect.element('@title').to.be.present.before(browser.globals.elementTimeout);
+  },
+  'Clinic workspace selection'(browser) {
+    const clinicWorkspacePage = browser.page.clinicWorkspacePage();
+    const clinicWorkspace = clinicWorkspacePage.section.clinicWorkspace;
+    const clinicPatientListPage = browser.page.clinicPatientListPage();
+    const clinicPatientList = clinicPatientListPage.section.patientList;
+    clinicWorkspace.waitForElementVisible('@title', browser.globals.elementTimeout);
+    clinicWorkspace.click('@goToWorkspace');
+    clinicPatientList.expect.element('@showAll').to.be.present.before(browser.globals.elementTimeout);
+  },
+  'Clinic workspace search custodial user and click patient name'(browser) {
+    const clinicPatientListPage = browser.page.clinicPatientListPage();
+    const clinicPatientList = clinicPatientListPage.section.patientList;
+    clinicPatientList.patientFilterSearch('custodial user');
+    clinicPatientList.waitForElementVisible('@row0PatientName', browser.globals.elementTimeout);
+    clinicPatientList.click('@row0PatientName');
+    clinicPatientList.expect.element('@uploadButton').to.be.present.before(browser.globals.elementTimeout);
+    clinicPatientList.expect.element('@resendVerificationEmailButton').to.be.present.before(browser.globals.elementTimeout);
+  },
+  'Click custodial user profile link'(browser) {
+    const userProfilePage = browser.page.userProfilePage();
+    const userProfile = userProfilePage.section.profile;
+    userProfile.waitForElementVisible('@profile', browser.globals.elementTimeout);
+    userProfile.click('@profile');
+    userProfile.expect.element('@edit').to.be.present.before(browser.globals.elementTimeout);
+  },
+  'Click export units and file type'(browser) {
+    const userProfilePage = browser.page.userProfilePage();
+    const userProfile = userProfilePage.section.export;
+    userProfile.waitForElementVisible('@mmoll', browser.globals.elementTimeout);
+    userProfile.click('@mmoll');
+    userProfile.waitForElementVisible('@json', browser.globals.elementTimeout);
+    userProfile.click('@json');
+    userProfile.expect.element('@mmoll').to.be.selected;
+    userProfile.expect.element('@json').to.be.selected;
+  },
+  'Click export in custodial user profile page | date range: all data': async (browser) => {
+    const userProfilePage = browser.page.userProfilePage();
+    const userProfile = userProfilePage.section.export;
+    userProfile.click('@allData');
+    userProfile.expect.element('@startDate').to.have.css('background-color').which.equals('rgba(204, 204, 204, 1)');
```

Comment:
**suggestion:** I think that better would be to check if input is `disabled`.
<img width="553" alt="image" src="https://github.com/user-attachments/assets/8e541a6a-bbc2-47a0-a17b-bc8c2443b45d">


Also you should check it on `@endDate` as well

Reply:
addressed in commit 9ec11bebda2ccc277206c30b7c350e0481e717dd